Frameworks/UIKit/UIViewController.mm

```diff
@@ -2245,7 +2253,6 @@ - (void)dealloc {
 
     priv->_childViewControllers = nil;
     priv->_searchDisplayController = nil;
-    priv->_storyboard = nil;
```

Comment:
...although I see that it's now a StrongId and therefore assignment to nil is unnecessary.

---
In reply to: [89896831](https://github.com/Microsoft/WinObjC/pull/1464#discussion_r89896831) [](ancestors = 89896831)